### ISSUE TITLE
Cross Entropy Loss —— 交叉熵Loss | 慕湮

### ISSUE THREAD

**Issue by tycallen** [tycallen/gitment-comments#11]
http://muyaan.com/2018/09/02/Cross-Entropy-Loss-%E2%80%94%E2%80%94-%E4%BA%A4%E5%8F%89%E7%86%B5Loss/
